pkg/proc/variables.go

```diff
@@ -690,7 +690,7 @@ func newVariable(name string, addr uint64, dwarfType godwarf.Type, bi *BinaryInf
 		v.Kind = reflect.Array
 		v.Base = v.Addr
 		v.Len = t.Count
-		v.Cap = -1
```

Comment:
Why are we changing this?

Reply:
I thought it was more correct to record the actual capacity of an array rather than -1. Go doesn't allow us to call `cap()` on a string, but it does allow it on an array, so I thought it'd be cleaner to have it.

Alternatively we can give reslicing a special case for arrays too, but this felt cleaner in the long run.

Let me know if you'd prefer this to remain -1 and I'll change reslice instead.

Reply:
Ok, I think it's fine to change it.